Scripts/script-ParseCSV.yml

```diff
@@ -65,23 +65,23 @@ script: |
               next(csvdata)
 
           for row in csvdata:
-              if parseip:
+              if parseip is not None:
```

Comment:
Just need to make sure this works from playbook task because playbook tasks might pass parameters as empty string ('') and so this will be true even if user did not specify.